File: fish-spots/src/model/dto/OutingDto.ts
export interface OutingDto {
    readonly outingId?: number;
    readonly locationId: number;
    readonly username: string;
    readonly outingDate: Date;
    readonly startTime: string;
    readonly endTime: string;
    readonly notes?: string;
    readonly createdAt?: Date;
    readonly lastUpdatedAt?: Date;
}

File: backend/FishSpots/FishSpots.Domain/RequestModels/OutingDto.cs
namespace FishSpots.Domain.RequestModels
{
    public class OutingDto
    {
        public int LocationId { get; set; }

        public required DateTime OutingDate { get; set; }

        public string Notes { get; set; }

        public string StartTime { get; set; }

        public string EndTime { get; set; }
    }
}
